$MirOS: ports/lang/egcs/gcc4.4/patches/patch-gcc_doc_gcc_1,v 1.1 2009/12/06 15:52:19 tg Exp $
--- gcc/doc/gcc.1.orig	Tue Mar 13 10:10:39 2012
+++ gcc/doc/gcc.1	Sun Dec  1 04:10:11 2013
@@ -123,17 +123,17 @@
 .rm #[ #] #H #V #F C
 .\" ========================================================================
 .\"
-.IX Title "GCC 1"
-.TH GCC 1 "2012-03-13" "gcc-4.4.7" "GNU"
+.IX Title "GCC-VSS 1"
+.TH GCC\-VSS 1 "2012-03-13" "gcc-4.4.7" "GNU"
 .\" For nroff, turn off justification.  Always turn off hyphenation; it makes
 .\" way too many mistakes in technical documents.
 .if n .ad l
 .nh
 .SH "NAME"
-gcc \- GNU project C and C++ compiler
+gcc\-VSS \- GNU project C and C++ compiler
 .SH "SYNOPSIS"
 .IX Header "SYNOPSIS"
-gcc [\fB\-c\fR|\fB\-S\fR|\fB\-E\fR] [\fB\-std=\fR\fIstandard\fR]
+gcc\-VSS [\fB\-c\fR|\fB\-S\fR|\fB\-E\fR] [\fB\-std=\fR\fIstandard\fR]
     [\fB\-g\fR] [\fB\-pg\fR] [\fB\-O\fR\fIlevel\fR]
     [\fB\-W\fR\fIwarn\fR...] [\fB\-pedantic\fR]
     [\fB\-I\fR\fIdir\fR...] [\fB\-L\fR\fIdir\fR...]
